webpack.config.babel.js

```diff
@@ -33,7 +35,15 @@ export default {
 				loader: 'vue-loader',
 				options: {
 					loaders: {
-						js: 'babel-loader',
+						js: {
```

Comment:
This is one of the bits I'm not so sure about,

Reply:
IIRC I had to add this to get it to work... but now not sure why or exactly what the problems were.

Reply:
I was getting lots of errors when referencing a definitions folder stored outside of node_modules.